src/testing/time_refinement_study_advection.cpp

```diff
@@ -0,0 +1,91 @@
+#include "time_refinement_study_advection.h"
```

Comment:
rename .cpp/.h files to be consistent with the class name

Reply:
Renaming to `time_refinement_study` in next commit.